bip-0032.mediawiki

```diff
@@ -253,29 +253,59 @@ Seed (hex): fffcf9f6f3f0edeae7e4e1dedbd8d5d2cfccc9c6c3c0bdbab7b4b1aeaba8a5a29f9c
 
 ==Implementations==
 
-Two Python implementations exist:
+===C===
 
-PyCoin (https://github.com/richardkiss/pycoin) is a suite of utilities for dealing with Bitcoin that includes BIP0032 wallet features.  BIP32Utils (https://github.com/jmcorgan/bip32utils) is a library and command line interface specifically focused on BIP0032 wallets and scripting.
+* [https://github.com/libbtc/libbtc libbtc] is a simple library for Bitcoin Wallets that implements BIP32.
 
-A Java implementation is available at https://github.com/bitsofproof/supernode/blob/1.1/api/src/main/java/com/bitsofproof/supernode/api/ExtendedKey.java
+===C++===
 
-A C++ implementation is available at https://github.com/ciphrex/mSIGNA/blob/master/deps/CoinCore/src/hdkeys.h
+* [https://github.com/ciphrex/mSIGNA mSIGNA] is an interactive desktop application for transacting on the bitcoin network, with a sample implementation available at https://github.com/ciphrex/mSIGNA/blob/master/deps/CoinCore/src/hdkeys.h
 
-An Objective-C implementation is available at https://github.com/oleganza/CoreBitcoin/blob/master/CoreBitcoin/BTCKeychain.h
+===C#===
 
-A Ruby implementation is available at https://github.com/GemHQ/money-tree
+* [https://github.com/NicolasDorier/NBitcoin NBitcoin] is a Bitcoin library for the .NET platform which implements BIP32.
 
-Two Go implementations exist:
+===Golang===
 
-hdkeychain (https://github.com/conformal/btcutil/tree/master/hdkeychain) provides an API for bitcoin hierarchical deterministic extended keys (BIP0032).  Go HD Wallet (https://github.com/WeMeetAgain/go-hdwallet).
+* [https://github.com/conformal/btcutil/tree/master/hdkeychain hdkeychain] provides an API for Bitcoin hierarchical deterministic extended keys.
 
-Two JavaScript implementations exist: available at https://github.com/sarchar/brainwallet.github.com/tree/bip32 and https://github.com/bitpay/bitcore
+* [https://github.com/WeMeetAgain/go-hdwallet Go HD Wallet tools] is a suite of Go utilities for generating hierarchical deterministic Bitcoin wallets.
 
-A PHP implemetation is available at https://github.com/Bit-Wasp/bitcoin-lib-php
+===Haskell===
 
-A C# implementation is available at https://github.com/NicolasDorier/NBitcoin (ExtKey, ExtPubKey)
+* [https://github.com/haskoin/haskoin Haskoin] is an implementation of the Bitcoin protocol in Haskell which implements BIP32 (in haskoin-core).
 
-A Haskell implementation is available at https://github.com/haskoin/haskoin together with a CLI interface at https://github.com/np/hx
+===Java===
+
+* [https://github.com/bitcoinj/bitcoinj bitcoinj] is a Java implementation of the Bitcoin protocol which implements BIP 32.
+
+* A sample implementation is available at https://github.com/bitsofproof/supernode/blob/1.1/api/src/main/java/com/bitsofproof/supernode/api/ExtendedKey.java
+
+===JavaScript===
+
+* [https://github.com/bitcoinjs/bip32-utils bip32-utils] is a set of utilities for working with BIP32.
```

Comment:
As the author of this library,  the actual underlying BIP32 code is currently in [`bitcoinjs-lib`](https://github.com/bitcoinjs/bitcoinjs-lib),  this will change soon as we merge it all into `bip32` as a standalone module.